website/_config.yml

```diff
@@ -28,6 +28,12 @@ nav_external_links:
     url: https://github.com/PicnicSupermarket/error-prone-support
     hide_icon: false
 
+callouts:
+  summary:
+    color: blue
+  note:
+    color: grey-dk
```

Comment:
Just checking: this will only be used in the next PR, right?

Reply:
Correct. All changes in this PR are not effective on `master` yet but are (almost all -- cases where we didn't, I tested locally against the `website` branch) already present on the `website` branch and therefore deployed. After @rickie's docgen modules, I'll open a PR to move the Ruby templating logic and a subsequent PR to change the GHA logic.